apps/dashboard/src/@/components/ui/tabs.tsx

```diff
@@ -50,7 +50,7 @@ export function TabLinks(props: {
         {/* Active line */}
         <div
           ref={lineRef}
-          className="absolute left-0 bottom-0 z-10 h-[2px] bg-foreground rounded-lg fade-in-0 animate-in"
```

Comment:
Good catch, also added in https://github.com/thirdweb-dev/js/pull/3829/files

Reply:
For some reason - vercel is not including this change in build and I'm also not getting it locally and there's this warning from graphite 

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/b9d7e6d2-8a7f-4f60-a027-b6274eaaaaa7">

Reply:
I did `gt track` and that fixed this warning, bu still not getting this particular change locally

Reply:
looks like same in happening in vercel deployment, it is not including this change